src/init.ts

```diff
@@ -45,8 +43,7 @@ async function query(
   return answers.query;
 }
 
-async function generatePackageJson(
-    packageJson: any, options: Options): Promise<void> {
+async function addScripts(packageJson: any, options: Options): Promise<boolean> {
```

Comment:
Consider:

```ts
function doSomthing(): boolean {
 /* forgot to return */
}

function doAnotherThing {
 /* forgot to return */
}

const didSomething  = doSomething();
const didAnotherThing = doAnothingThing();
```

You get a compiler error for `doSomething` but not for `doAnotherThing`. This is an easy mistake to make and I actually made the mistake as I was working on this PR.

I am trying to find a reference in the TS Style guide that mandates that return types are required – for the JS style guide at least [they are required](https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md?cl=head#features-functions-returns).

I wonder if tslint can enforce this. It does not seem to be doing so presently.